### DISCUSSION TITLE
Continuation of supporting Hydra

### DISCUSSION THREAD

**Discussion by ckipp01** [scalacenter/bloop#1970] [General] [answered]
So inside of the Bloop code base there are certain parts of the code that are dedicated to ensuring that [Hydra](https://www.triplequote.com/) is handled correctly. For example sometimes entire classes like the [ConcurrentAnalysisCallback](https://github.com/scalacenter/bloop/blob/main/backend/src/main/scala/sbt/internal/inc/bloop/internal/ConcurrentAnalysisCallback.scala). In the past we've tested against Hydra for some stuff, but everything is ignored now in the `HydraCompileSpec`. Since we've been doing quite a bit of cleanup and updating the following questions pop up:

- Do we need to keep all these tests in here that are just currently ignored?
- How often is Bloop used with Hydra? Is there a specific need for this, or is was this a "nice to have"?
- Can we just rip all the Hydra stuff out depending on the above?

**Comment by Duhemm** [comment, marked as answer]
I haven't checked, but it could be that the license has expired. I forwarded to you the last discussion I had with @dotta regarding Bloop's Hydra license.

**Reply by ckipp01** [reply]
Awesome, thanks @Duhemm! I'll follow-up with them.

**Reply by tgodzik** [reply]
It did expire and was a bit of a problem since we had to ping them every year. And I didn't get any response on the PR that ignored the tests https://github.com/scalacenter/bloop/pull/1619

**Reply by ckipp01** [reply]
Just a heads up, I've got a license! I just need to find some time and add it and re-enable stuff 😅 . I'll try to look in it this next week.
